docs/serving/services/configure-requests-limits-services.md

```diff
@@ -23,6 +23,48 @@ spec:
               cpu: 1
 ```
 
-## Additional resources
+## Configure Queue Proxy resources
+
+In order to set the Queue Proxy resource requests and limits you can either
+set them globally in the [deployment config map](../configuration/deployment.md) or you can set them at the service level using the corresponding annotations targeting cpu, memory and ephemeral-storage resource types. The previous example becomes:
```

Comment:
```suggestion
set them globally in the [deployment config map](../configuration/deployment.md) or you can set them at the service level using the corresponding annotations targeting cpu, memory and ephemeral-storage resource types. The latter example becomes:
```

Sorry my English might be wrong, but "the previous" means the global setting? If so, here should be "the latter"?

Reply:
Yeah I am not sure either.

Reply:
maybe just say `above`